git-cliff-core/src/config.rs

```diff
@@ -109,6 +109,9 @@ pub struct GitConfig {
 	/// Regex to ignore matched tags.
 	#[serde(with = "serde_regex", default)]
 	pub ignore_tags:              Option<Regex>,
+	/// Regex to count matched tags.
+	#[serde(with = "serde_regex", default)]
+	pub count_tags:               Option<Regex>,
```

Comment:
Naming is hard. I name it "count" because it's if "counted".

"include" may be an alternative, but .. we have two aspects here: (1) whether count the commit (2) whether count the tag.